dashboard/app/views/shared/_extra_links.html.haml

```diff
@@ -1,6 +1,6 @@
 .extra-links
   %h4
-    Extra Links:
+    %span{onclick: 'return extraLinksDismissClick()'} Extra Links:
```

Comment:
What is this change doing?

Reply:
Hah, this was a little convenience that I'll remove from this PR.  It allowed the viewer to dismiss the admin menu in the bottom-left corner by clicking on its "Extra links:" text. :)